src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMessage.cs

```diff
@@ -183,48 +183,49 @@ private static byte[] DecodeSignature(CborReader reader)
             return reader.ReadByteString();
         }
 
-        internal static int CreateHashedToBeSigned(Span<byte> destination, string context, ReadOnlySpan<byte> encodedProtectedHeader, ReadOnlySpan<byte> content, HashAlgorithmName hashAlgorithm)
+        // TODO: reanme appendtobesigned
```

Comment:
TODO-rename on a non-public member seems weird...  why not either rename it, or remove the comment?

Reply:
Forgot to address the TODO, sorry.